overops-java-bullshifier/values.yaml

```diff
@@ -0,0 +1,107 @@
+# Default values for overops-java-bullshifier.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+# Overops Config settings
+overops:
+  collectorHost: ""
+  collectorPort: 6060
+  applicationName: ""
+  deploymentName: ""
+  serverName: ""
+  color: "white"
+  noAgent: false # Use for tests
+
+  # Use an agent-sidecard instead of an embedded v4 agent (https://github.com/takipi-field/kubernetes/tree/master/agent/sidecar).
+  agentSidecar:
+    enabled: false
+    useDebug: false
+    repository: overops/agent-sidecar
+    pullPolicy: IfNotPresent
+    tag: latest
+
+  # Job Related settings
+  job:
+    enabled: false
+    runnindDurationHours: 0
+    runnindDurationMinutes: 15
+
+replicaCount: 1
+
+image:
+  repository: overops-java-bullshifier-v4
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: latest
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+serviceAccount:
+  # Specifies whether a service account should be created
+  create: true
```

Comment:
I had this set to false for the event generator, we can do that for the bullishier too. Not sure theres a need but kept the template around if we needed it down the road.

Reply:
Done